cfme/networks/balancer.py

```diff
@@ -0,0 +1,84 @@
+from utils import version
+from utils.appliance.implementations.ui import navigator, CFMENavigateStep, navigate_to
+from navmazing import NavigateToSibling, NavigateToAttribute
+from utils.appliance import Navigatable
+from utils.update import Updateable
+from cfme.common import Taggable, SummaryMixin
+from cfme.networks.views import BalancerView
+from cfme.networks.views import BalancerDetailsView
```

Comment:
**REQUIRED**
Here and below imports must be done like this:
```python
from navmazing import NavigateToAttribute, NavigateToSibling

from cfme.common import SummaryMixin, Taggable
from cfme.networks.views import BalancerDetailsView, BalancerView
from utils import version
from utils.appliance import Navigatable
from utils.appliance.implementations.ui import CFMENavigateStep, navigator, navigate_to
from utils.update import Updateable
```
http://cfme-tests.readthedocs.io/guides/dev_guide.html#code-style

> In addition to being broken up into the three sections of standard library, third-party, and the local application, imports should be sorted alphabetically. ‘import’ lines within those sections still come before ‘from ... import’ lines